### PR TITLE
Feature/Migrate to Smart Wallet: intro screen

### PR DESCRIPTION
**Pull request by mdjastrzebski**
Scope:
- [x] Intro screen

### CODE REVIEW

File: src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+/* eslint-disable i18next/no-literal-string */
```
Comment (wjkg) [inline src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js:20]: Would there be that many exceptions to be added inline that it's necessary? The rule won't be of much use if it's being disabled for entire files.

File: src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+          <Spacing h={spacing.extraLarge} />
+
+          <Body>{t('footer')}</Body>
+
+          <Spacing h={48} />
+
+          <Button title={t('migrate')} onPress={navigateToMigration} />
+
+          <Spacing h={spacing.large} />
```
Comment (wjkg) [inline src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js:91]: It's inside a styled view, why not use that for padding?

**Review by wjkg — COMMENTED** [review]
Looks fine to me, I've marked a few minor things.

One small bug when opening the screen: on iOS, when using the light theme for the rest of the app, the status bar disappears, since it becomes black text on black background.

File: src/locales/en/common.json [hunk 3]
```diff
@@ -635,6 +635,21 @@
...
+      "header": "Your account is being upgraded to a Pillar Smart Wallet.",
+      "upgrade": {
+        "subtitle": "What is this upgrade?",
+        "body": "In a few words: Pillar is making Ethereum more accessible and laying a foundation for further improvements of Pillar platform."
+      },
+      "benefits": {
+        "one": "Benefit number one",
+        "two": "Benefit number two is long enought to fit in two lines",
+        "three": "This third benefit is rather long and all of a sudden requires three lines for copy"
+      },
```
[the remark this answers is not in the document]
Comment (mdjastrzebski) [reply src/locales/en/common.json:650]: Still waiting for final copy from marketing ;-)

File: src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+          <Logo source={smartWalletImage} />
+
+          <Subtitle>{t('upgrade.subtitle')}</Subtitle>
+          <Body>{t('upgrade.body')}</Body>
+
+          <Spacing h={spacing.medium} />
+
+          <Card title={t('benefits.one')} emoji="💸" />
+          <Card title={t('benefits.two')} emoji="✌️" />
+          <Card title={t('benefits.three')} emoji="😇" />
```
[the remark this answers is not in the document]
Comment (mdjastrzebski) [reply src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferIntro/KeyBasedAssetTransferIntro.js:81]: IMO react native emoji seems a bit overkill if we can pass emoji directly.

File: src/locales/en/common.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (wjkg) [reply src/locales/en/common.json:650]: What's the plan for merging, then?

[the remark this answers is not in the document]
Comment (mdjastrzebski) [reply src/locales/en/common.json:650]: Still waiting for marketing. I'll comment out the display section in the component code for now.

**Review by wjkg — APPROVED** [review]

**Pull request merged**